src/xdocs/config_coding.xml

```diff
@@ -2895,7 +2895,7 @@ public void myTest() {
               <td>subset of tokens
                 <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html">TokenTypes</a>
               </td>
-              <td><code>no tokens</code></td>
```

Comment:
can we remove support for no tokens from our validator?
to make sure we will not do this problem in design again.

```
src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java
1140:                        result = "no tokens";

```

Reply:
removed